test/index.spec.js

```diff
@@ -70,60 +66,80 @@ function runTestWithConfig (testConfig, label) {
             return state
         }
       }
+
+      // testConfig.debug = true;
       mockUndoableReducer = undoable(countReducer, testConfig)
+      store = Redux.createStore(mockUndoableReducer, initialStoreState)
```

Comment:
I've moved the store calls from below up to here since that is main way of using the initialState now.